test/helpers/index.js

```diff
@@ -10,10 +10,10 @@ const print = require('./print')
 const Server = require('./test-bot')
 
 module.exports = {
-  GroupId: () => `%${new Secret().toString()}.cloaked`,
-  GroupKey: () => new Secret().toBuffer(),
-  FeedId: () => new FeedId().mock().toTFK(),
-  PrevMsgId: () => new MsgId().mock().toTFK(),
+  GroupId: () => `%${new SecretKey().toString()}.cloaked`,
+  GroupKey: () => new SecretKey().toBuffer(),
+  FeedId: () => new FeedId().mock().toSSB(),
+  MsgId: () => new MsgId().mock().toSSB(),
```

Comment:
I like this change - everything that is exported is now in the same format (ssb strings)